backend/question-service/src/controllers/questionController.ts

```diff
@@ -433,18 +440,27 @@ const formatQuestionResponse = (question: IQuestion) => {
   };
 };
 
-const formatQuestionIndivResponse = (
+const formatQuestionIndivResponse = async (
   question: IQuestion,
   questionTemplate: IQuestionTemplate,
 ) => {
+  const testcaseDelimiter = "\n";
```

Comment:
change to "\n\n" to separate test cases?, since "\n" is used to separate variables in a test case